### PR TITLE
norduser grpc server

### PR DESCRIPTION
**Pull request by bartoszWojciechO**

### CODE REVIEW

File: cmd/norduser/main.go [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+		log.Fatalf("Error on listening to UNIX domain socket: %s\n", err)
+	}
+
+	server := norduser.NewServer()
+
+	grpcServer := grpc.NewServer()
+	pb.RegisterNorduserServer(grpcServer, server)
+
+	go func() {
+		if err := grpcServer.Serve(listener); err != nil {
```
Comment (mariusSincovici) [inline cmd/norduser/main.go:34]: If this fails, does it make sense for the app to continue running?

File: norduser/service/fork.go [hunk 2]
```diff
@@ -25,8 +27,37 @@ type ChildProcessNorduser struct {
 	logFile io.Closer
 }
 
+func isRunning(uid uint32) (bool, error) {
+	debug.PrintStack()
```
Comment (mariusSincovici) [inline norduser/service/fork.go:31]: is it needed?

File: norduser/service/fork.go [hunk 3]
```diff
@@ -25,8 +27,37 @@ type ChildProcessNorduser struct {
...
+
+	// list all norduserd processes, restrict output to uid of the owner
+	output, err := exec.Command("ps", "-C", internal.Norduserd, "-o", "uid=").CombinedOutput()
+	if err != nil {
+		fmt.Println("listing processes: %w", err)
```
Comment (mariusSincovici) [inline norduser/service/fork.go:36]: shouldn't here be an error returned?

File: cmd/norduser/main.go (hunk 1 above)
Comment (bartoszWojciechO) [reply cmd/norduser/main.go:34]: `log.Fatalln` will terminate the program.

**Review by mariusSincovici — APPROVED** [review]

**Review by keliramu — APPROVED** [review]

**Pull request merged**